.github/workflows/security-action.yml

```diff
@@ -0,0 +1,32 @@
+name: security
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches: [main]
+
+jobs:
+  security:
+    name: security
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      # CodeQL analyzed languages
+      matrix:
+        language: [ 'generic', 'javascript', 'python', 'ruby', 'go' ]
```

Comment:
cc @thypon is there any expected issues by adding go here? This is just copied from the example in the security-actions repo otherwise

Reply:
Should be fine, CodeQL [supports](https://codeql.github.com/docs/codeql-overview/supported-languages-and-frameworks/) go.